.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -1,5 +1,17 @@
 name: rawls-run-azure-e2e-tests
 
+# This workflow can be run on schedule OR dispatched (manually or by another dispatcher workflow)
+# When thw workflow is triggered on schedule, it will use the parameters specified on
+# the RHS of || in init-github-context.extract-inputs step. Otherwise, if the workflow is
+# triggered by a dispatch event, it will use the parameters specified on LHS of || in
+# the init-github-context.extract-inputs step.
+#
+# Ex.
+#
+# echo "student-subjects=${{ toJson(inputs.student-subjects || '["harry.potter@quality.firecloud.org","lavender.brown@quality.firecloud.org"]') }}" >> "$GITHUB_OUTPUT"
```

Comment:
I'm not clear yet why we need to be able to specify the values different for a schedule vs. dispatch. Can you explain what the use case is?

Reply:
You see the same pattern with `branch` and `delete-bee`. 

```
          echo "branch=${{ inputs.branch                            || 'develop' }}" >> "$GITHUB_OUTPUT"
          echo "delete-bee=${{ inputs.delete-bee                    || false }}" >> "$GITHUB_OUTPUT"
```

Scheduled workflows are configured to run at specific times or intervals `without user interaction`, and they do not provide a mechanism for users to input values during scheduling.

Inputs are commonly associated with workflows triggered by events, pull requests, or dispatch actions, where users can specify inputs when triggering the workflow.

To enable the use of the same code for both scheduled and manual (dispatched) runs, we have adopted an LHS/RHS parsing approach. Otherwise we will have to do something below which doesn't look as straightforward as doing the same thing in a single line of code.

```
if [[ -n "${{ inputs.branch }}" ]]; then
  echo "branch=${{ inputs.branch }}" >> "$GITHUB_OUTPUT"
else
  echo "branch=develop" >> "$GITHUB_OUTPUT"
fi
```

Reply:
Ok, I see.